app/src/main/java/com/example/keybindhelper/ui/Projects/ProjectsFragment.kt

```diff
@@ -88,11 +88,11 @@ class ProjectsFragment : Fragment() {
     fun openKeybindFragment(){
         NavHostFragment.findNavController(this).navigate(R.id.nav_keybind)
     }
-    fun RefreshProjectList():List<Project>{
+    fun refreshProjectList():List<Project>{
```

Comment:
Its better practice to use camelcase for naming these.
source: https://kotlinlang.org/docs/coding-conventions.html

Reply:
Good, but covered in a previous code review.